### ISSUE TITLE
Update Yejin's Korea-Yonsei 학술교류 발표회 

### ISSUE THREAD

**Issue by d-h-lee** [AIML-K/aiml-k.github.io#83]
- Need QR code for this poster presentation
- (Later) add photo of the event

**Comment by d-h-lee** [comment]
Related event: https://github.com/AIML-K/bamboo-forest/issues/183

**Comment by d-h-lee** [comment]
Added the photo and the poster. See https://aiml-k.github.io/publication/2024koreayonsei-science-jeong-etal/

@JeongYejin1226 do you think adding abstract would be useful here? let me know.

**Comment by JeongYejin1226** [comment]
Yes, I think it would be helpful. I’m attaching the abstract of the paper.

> Abstract
> Background context: Accurate and consistent measurement of sagittal alignment is challenging, particularly in patients with severe coronal deformities, including degenerative lumbar scoliosis (DLS). 
> Purpose: This study aimed to develop and validate an artificial intelligence (AI)-based system for automating the measurement of key sagittal parameters, including lumbar lordosis, pelvic incidence, pelvic tilt, and sacral slope, with a focus on its applicability across a wide range of deformities, including severe coronal deformities, such as DLS.
> Design: Retrospective observational study.
> Patient sample: A total of 1,011 standing lumbar lateral radiographs, including DLS.
> Outcome measure: Interclass and intraclass correlation coefficients (CC), and Bland-Altman plots.
> Methods: The model utilizes a deep-learning framework, incorporating a U-Net for segmentation and a Keypoint Region-based Convolutional Neural Network for keypoint detection. The ground truth masks were annotated by an experienced orthopedic specialist. The performance of the model was evaluated against ground truth measurements and assessments from two expert raters using interclass and intraclass CC, and Bland-Altman plots.
> Results: In the test set of 113 patients, 39 (34.5%) had DLS, with a mean Cobb’s angle of 14.8° ± 4.4°. The AI model achieved an intraclass CC of 1.00 across all parameters, indicating perfect consistency. Interclass CCs comparing the AI model to ground truth ranged from 0.96 to 0.99, outperforming experienced orthopedic surgeons. Bland-Altman analysis revealed no significant systemic bias, with most differences falling within clinically acceptable ranges. A 5-fold cross-validation further demonstrated robust performance, with interclass CCs ranging from 0.96 to 0.99 across diverse subsets.
> Conclusion: This AI-based system offers a reliable and efficient automated measurement of sagittal parameters in spinal deformities, including severe coronal deformities. The superior performance of the model compared with that of expert raters highlights its potential for clinical applications.
> 
> Keywords: Pelvic parameter, convolutional neural network, sagittal alignment, artificial intelligence, coronal deformity, segmentation, keypoint detection.

**Comment by d-h-lee** [comment]
This needs to be updated to reflect the actual journal article: https://www.sciencedirect.com/science/article/abs/pii/S1529943025000531
